packages/contracts/contracts/plugin/PluginSetupProcessor.sol

```diff
@@ -32,27 +33,63 @@ contract PluginSetupProcessor is DaoAuthorizable {
     bytes32 public constant APPLY_UNINSTALLATION_PERMISSION_ID =
         keccak256("APPLY_UNINSTALLATION_PERMISSION");
 
-    struct PluginUpdateParams {
-        address plugin; // The address of the `Plugin` contract being currently active. This can be a proxy or a concrete instance.
-        PluginRepo pluginSetupRepo; // The repository storing the `PluginSetup` contracts of all versions of a plugin.
-        address currentPluginSetup; // The `PluginSetup` contract of the version being currently active and to be updated from.
-        address newPluginSetup; // The `PluginSetup` contract of the version to be updated to.
+    /// @notice Used when there's UI update only which gets stored for the setupId generation.
+    /// @dev keccak256(abi.encode[])
+    bytes32 private constant EMPTY_ARRAY_ENCODED_HASH =
+        bytes32(0x569e75fc77c1a856f6daaf9e69d8a9566ca34aa47f9133711ce065a571af0cfd);
```

Comment:
why 
`bytes32 private constant EMPTY_ARRAY_ENCODED_HASH =
        bytes32(0x569e75fc77c1a856f6daaf9e69d8a9566ca34aa47f9133711ce065a571af0cfd);`
and not
`bytes32 private constant EMPTY_ARRAY_ENCODED_HASH =
        0x569e75fc77c1a856f6daaf9e69d8a9566ca34aa47f9133711ce065a571af0cfd;`
?